subprojects/core/src/main/java/org/gradle/internal/build/BuildState.java

```diff
@@ -69,6 +69,11 @@ public interface BuildState {
      */
     boolean isProjectsLoaded();
 
+    /**
+     * Is the mutable model for projects available?
+     */
+    boolean isProjectsAvailable();
```

Comment:
🤔 The name seems to be too generic, and reading the javadoc is required to understand the meaning.
How about `isProjectsMutable()` or `isProjectsAvailableForMutation()` ?
An alternativ to keep the naming consistent with `ProjectState.isCreated()` and go with `isProjectsCreated()`

Reply:
I'd go with `isProjectsCreated` to keep it consistent with `ProjectState#isCreated` and even include a `@see` link.

Reply:
👍